spikeextractors/extractorlist.py

```diff
@@ -34,13 +35,15 @@
 from .extractors.alfsortingextractor import ALFSortingExtractor
 from .extractors.cedextractors import CEDRecordingExtractor
 from .extractors.cellexplorersortingextractor import CellExplorerSortingExtractor
+from .extractors.neuropixelsdatrecordingextractor import NeuropixelsDatRecordingExtractor
 
 recording_extractor_full_list = [
     MdaRecordingExtractor,
     MEArecRecordingExtractor,
     BiocamRecordingExtractor,
     ExdirRecordingExtractor,
     OpenEphysRecordingExtractor,
+    OpenEphysNPIXRecordingExtractor,
```

Comment:
The OpenEphys team is calling this their "Binary" format. I feel like we should follow their convention.